lib/bolt/error.rb

```diff
@@ -166,4 +166,10 @@ def initialize(msg, path)
       super(msg, 'bolt/file-error', { "path" => path })
     end
   end
+
+  class PluginNotSupported < Bolt::Error
```

Comment:
Let's put this in `lib/bolt-server` for now since there is currently no way for Bolt's plugin code to raise this error.